darts/metrics/metrics.py

```diff
@@ -6,43 +6,102 @@
 """
 
 import numpy as np
-from typing import Tuple
 from ..timeseries import TimeSeries
+from darts.utils import _parallel_apply, _build_tqdm_iterator
 from ..utils.statistics import check_seasonality
-from ..logging import raise_if_not, get_logger
+from ..logging import raise_if_not, get_logger, raise_log
 from warnings import warn
-from typing import Optional, Callable
+from typing import Optional, Callable, Sequence, Union, Tuple
 from inspect import signature
 from functools import wraps
 
+
 logger = get_logger(__name__)
 
 
+"""
+Note: for new metrics added to this module to be able to leverage the two decorators, it is required both having
+the `actual_series` and `pred_series` parameters, and not having other `Sequence` as args (since these decorators
+don't “unpack“ parameters different from `actual_series` and `pred_series`). In those cases, the new metric must take
+care of dealing with Sequence[TimeSeries] and multivariate TimeSeries on its own (See mase() implementation).
+"""
+
+
+def multi_ts_support(func):
+    """
+    This decorator further adapts the metrics that took as input two univariate/multivariate `TimeSeries` instances,
+    adding support for equally-sized array of `TimeSeries` instances. The decorator computes the pairwise metric for
+    `TimeSeries` with the same indices, and returns a float value that is computed as a function of all the
+    pairwise metrics using a `inter_reduction` subroutine passed as argument to the metric function.
+
+    If a 'Sequence[TimeSeries]' is passed as input, this decorator provides also parallelisation of the metric
+    evaluation regarding different `TimeSeries` (if the `n_jobs` parameter is not set 1).
+    """
+
+    @wraps(func)
+    def wrapper_multi_ts_support(*args, **kwargs):
+        actual_series = kwargs['actual_series'] if 'actual_series' in kwargs else args[0]
+        pred_series = kwargs['pred_series'] if 'pred_series' in kwargs else args[0] if 'actual_series' in kwargs \
+            else args[1]
+
+        n_jobs = kwargs.pop('n_jobs', 1)
```

Comment:
```suggestion
        n_jobs = kwargs.pop('n_jobs', signature(func).parameters['n_jobs'].default)
        verbose = kwargs.pop('verbose', signature(func).parameters['verbose'].default)
```

Taking inspiration from the way the `reduction` parameter is set, this should fix the problem 👍🏼   Furthermore, I noticed that this "kwargs or default argument" approach could fail in the case these parameters are passed as positional args instead of kwargs. Thus I would suggest to add the `*` parameter in the metrics functions' signatures before the parameters we need to be passed as kwargs (`reduction`, `inter_reduction`, `n_jobs`, `verbose`).